clients/java/zts/pom.xml

```diff
@@ -28,6 +28,21 @@
 
   <dependencies>
     <dependency>
+      <groupId>com.amazonaws</groupId>
```

Comment:
this adds the full aws jdk as a dependency where we don't need most of it. check out how zts server handles this with the aws-java-sdk-bom and we should use the same version as zts server. 1.0.67 is quite old.